docs/dev_setup_osx.rst

```diff
@@ -175,7 +192,7 @@ In addition to WebStorm, you will also need to have the Dart SDK installed.  Ple
 
 **Note:** security_monkey is currently pinned to dart v1.12.1 and does not work with newer versions.  Until we fix that, you'll need to download the dart sdk manually at https://www.dartlang.org/downloads/archive/
```

Comment:
See PR #357 and specifically commit af46c2a

We removed the dart pin back in June.